handbook/company/product-groups.md

```diff
@@ -26,43 +26,58 @@ At Fleet, [anyone can contribute](https://fleetdm.com/handbook/company#openness)
 
 ## Current product groups
 
-| Product group                            | Goal _(value for customers and/or community)_                         | Capacity\*   |
-|:-----------------------------------------|:----------------------------------------------------------------------|:-------------|
-| [Endpoint ops](#endpoint-ops-group)      | Increase and exceed maturity in the "Endpoint operations" category.   | 156          |
-| [MDM](#mdm-group)                        | Reach maturity in the "MDM" product category.                         | 156          |
+| Product group                            | Goal _(value for customers and/or community)_                                                                          | Capacity |
+|:-----------------------------------------|:-----------------------------------------------------------------------------------------------------------------------|:---------|
+| [MDM](#mdm-group)                        | Increase and exceed maturity in the [device management](https://fleetdm.com/device-management) product category.       | 104      |
+| [Orchestration](#orchestration-group)    | Increase and exceed maturity in the [orchestration](https://fleetdm.com/orchestration) product category.               | 104      |
+| [Software](#software-group)              | Increase and exceed maturity in the [software management](https://fleetdm.com/software-management) product category.   | 104      |
```

Comment:
Time to build a MOSsy fleet :)

Also @sharon-fdm you cool with me referring to your new team as `Team Orca?` Orchestration has so many syllables :D

Reply:
Orca sounds really cool! 
![image](https://github.com/user-attachments/assets/55da0b76-3d04-4a08-8dbb-69fd3310df5a)